andela/AndelaEatsUI
# #160237207 Implement add vendor functionality

**What does the PR do?**
- Implements add vendor functionality

**Description of Task to be completed?**
- Ensures an admin can add a vendor

**How should this be manually tested?**
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git) and
-  cd into AndelaEatsUI and checkout to this branch `ft-implement-create-vendor-functionality-160237207`
- run `npm run bundle` and `npm run dev`
- Log in as an `Admin`, navigate to `admin dashboard`, select `Vendors`, and click on `Add Vendor`
- supply the required information and click `Add vendor button`
- run `npm test` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#160237207](https://www.pivotaltracker.com/story/show/160237207)

**Screenshots**
- N/A
